scripts/getting-started.sh

```diff
@@ -124,9 +124,8 @@ fi
 
 # Ensure that we have wasm support
 if prompt_default_yes "\n🦀 Setup the Rust environment (e.g. WASM support)?"; then
-    version=`curl -s -H "Accept:application/vnd.github.v3.raw" https://api.github.com/repos/paritytech/polkadot-sdk/contents/rust-toolchain.toml | grep 'channel =' | awk -F'"' '{print $2}'`
     echo "🦀 Setting up Rust environment."
-    rustup default $version
+    rustup default stable
```

Comment:
But this is still setting default toolchain :confused: 

This like should not be necessary, just like the rest of commands in this block, `rust-toolchain.toml` will take care of it all.

Reply:
I was so sure I fixed that... One moment

Reply:
Removed the entire block, as discussed